README.md

```diff
@@ -24,12 +24,12 @@ dex consists of multiple components:
 	- provides an API for bootstrapping the system.
 - **dexctl** is a CLI tool used to manage a dex deployment
 	- configure identity provider connectors
-	- administer OIDC client identities
+	- administer OIDC client identities.
```

Comment:
Don't know if the periods are necessary since these aren't complete sentences. We should also be consistent for all bullet points.

Deferring to @joshix for style preferences. The style guide in coreos/docs doesn't go over short bullet points like this.